client.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
```

Comment:
What is the purpose of this file?

Reply:
This is the code which talks to rancher.